### PR TITLE
Zwift.Zwift 1.0.52

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/22089)

### CODE REVIEW

**Comment by vedantmgoyal2009** [comment]
What is happening?

**Comment by vedantmgoyal2009** [comment]
Now good!

**Comment by ImJoakim** [comment]
> What is happening?

Happening for others too - https://github.com/microsoft/winget-pkgs/pull/21204#issuecomment-884748638

File: manifests/z/Zwift/Zwift/1.0.52/Zwift.Zwift.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: exe
+  Scope: user
+  InstallerUrl: https://cdn.zwift.com/app/ZwiftSetup.exe
+  InstallerSha256: 0FA151B1ADF8813E93EFFD8154CBF3E8437D4E7D78E8AAE29DFAAF451B2D2028
+  InstallerSwitches:
+    Silent: /VERYSILENT /NORESTART
+    SilentWithProgress: /SILENT /NORESTART
```
Comment (OfficialEsco) [inline manifests/z/Zwift/Zwift/1.0.52/Zwift.Zwift.installer.yaml:20]: Where did you get the user scope from?

```suggestion
  Architecture: x64
  InstallerType: inno
  Scope: machine
  InstallerUrl: https://cdn.zwift.com/app/ZwiftSetup.exe
  InstallerSha256: 0FA151B1ADF8813E93EFFD8154CBF3E8437D4E7D78E8AAE29DFAAF451B2D2028
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**